### PR TITLE
Add helper to do hashing

### PR DESCRIPTION
**Pull request by nwillems**
I made this small helper tool, dunno if its in line with how you like the repo. Feel free to reject it :-)

### CODE REVIEW

File: cmd/hasher.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	"fmt"
+	"os"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+func main() {
+	password := os.Args[1]
+
+	hash, err := bcrypt.GenerateFromPassword([]byte(password), 12)
```
Comment (decke) [inline cmd/hasher.go:13]: The hardcoded cost of 12 is higher than the default but I think in general we should stick to bcrypt.DefaultCost

Comment (nwillems) [reply cmd/hasher.go:13]: Nice. Didn't know of that.

Comment (decke) [reply cmd/hasher.go:13]: @nwillems: Can you please update your patch to use bcrypt.DefaultCost instead of 12?

Comment (nwillems) [reply cmd/hasher.go]: Should be done by now. Sorry for the delay.

**Comment by nwillems** [comment]
@decke do you want the commits squashed?

**Comment by decke** [comment]
Thanks for your contribution!

**Pull request merged**